openedx/features/edly/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+
+
+class Organization(models.Model):
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+    name = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+    slug = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
+
+    def __repr__(self):
+        return self.name
+
+
+class SubOrganization(models.Model):
+    alphanumeric = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+    name = models.CharField(max_length=100, unique=True, validators=[alphanumeric])
```

Comment:
remove name and slug field. organization model has `name` and `short_name`

Reply:
The fields naming was based on the following acceptance criteria:
`create edly organization model (edly_panel.apps.organizations.models.Organization) with name and slug (unique identifier only alphanumeric value is allowed)`

Kindly verify as per your suggestion now the names will be:

Organization Model:  `name` and `short_name`
SubOrganization Model: name` and `slug`

Reply:
In AC's I suggested to have name and slug for `EdlyOrganization` only.

Reply:
EdlySubOrganization should have 1-1 relation with edx organization and we can use the `name` and `short_name` of edx organization.

Reply:
> **1 edly-sub-organization : 1 edx organization : 1 partner in discovery service : 1 django site (LMS, Studio, ECOM, Credentials) : 1 WP site**

Kindly clarify this comment.

Reply:
Done